vlib/v/fmt/tests/struct_and_alias_comments_keep.vv

```diff
@@ -0,0 +1,13 @@
+module main
+
+pub struct CSDL_GamepadBinding {
+pub mut:
+	output_type u8
+	// foobar
+}
+
+// TODO: BELONGS ABOVE // union {
+//  SDL_GamepadButton button; struct {
+//  SDL_GamepadAxis axis; int axis_min; int axis_max; } axis; } output
```

Comment:
I am confused. Should not the test case be, with the comments inside the struct?

Reply:
The _keep.vv file here can be formatted as is on master too.

Reply:
Ouch, good point. I think bug messed up the test case. 👍🏻